openvpn.tf

```diff
@@ -27,7 +27,7 @@ locals {
 }
 
 module "openvpn" {
-  source = "github.com/cisagov/openvpn-server-tf-module"
+  source = "github.com/cisagov/openvpn-server-tf-module?ref=improvement%2Fseparate-trusted-ip-lists-for-ssh-and-vpn"
```

Comment:
Reminder: You'll want to remove this temporary testing tidbit...

Reply:
Yep, I forgot to remove that.  Glad you noticed it.  Removed in commit 0073279.